### DISCUSSION TITLE
Host data sets as service

### DISCUSSION THREAD

**Discussion by sebastianfrey** [maja601/EuroCrops#3] [Ideas]
Any plans to host those data sets as service?

**Comment by maja601** [comment]
Hi @sebastianfrey,
yes, we are currently working with some national and international data providers / services to make the datasets available via (probably) API and WMS. I'll post any updates regarding the releases here.

**Reply by sebastianfrey** [reply]
Cool. Can you clarify what you mean by `API`? `OGC API Features` or something different?

**Reply by maja601** [reply]
Haven't quite made my mind up on that, but I'll happily take suggestions into account :smile:

**Reply by sebastianfrey** [reply]
Got it, how you meant the term API.

Well, I think `OGC API Features` seems here like a good starting point, since it is the next big OGC standard for serving vector data over a simple and easy to use RESTful API. Alternatives that come to mind are WFS or ArcGIS, the former being legacy and the latter proprietary.

**Reply by maja601** [reply]
Thank you!! Yes, that is definitely the type of input I was hoping for. I'll let you know how we proceed!
